android/build.gradle

```diff
@@ -8,6 +8,7 @@ buildscript {
         compileSdkVersion = 31
         targetSdkVersion = 31
         ndkVersion = "25.1.8937393"
+        kotlinVersion = '1.7.10'
```

Comment:
As you can see there is a duplicate with `kotlinversion`, you can safely remove the line number 4 in this file.

Reply:
fixed